docs/NoSSL/lavalink-without-ssl.md

```diff
@@ -17,6 +17,24 @@ description: Without SSL should work out of the box!
     v4 introduce a breaking changes that affects all library that are using v3 API.
     You must update your bot library to support v4!
 
+### Hosted by @ [Caliwyr](https://discord.gg/6xpF6YqVDd)
+Version 4.0.4 (Plugin) | [Uptime Status](https://lavalink-status.oryzen.xyz/)
+```bash
+Host : lavalink-v4.oryzen.xyz
+Port : 80
+Password : "oryzen.xyz"
+Secure : false    
+```
+
+### Hosted by @ [Caliwyr](https://discord.gg/6xpF6YqVDd)
```

Comment:
Please remove this line as it is by the same author

Reply:
alright, i will remove it